app/workers/github_issue_worker.rb

```diff
@@ -0,0 +1,17 @@
+class GithubIssueWorker
+  include Sidekiq::Worker
+
+  def perform(payload)
+    parsed_payload = JSON.parse(payload)
```

Comment:
This worker is what process the github requests with the condition that it has to be a valid payload with an action and an issue

Reply:
this is pretty cool! 

the only downside I see with this, is that right now both the responsibility for processing the webhook (validating & knowing the data format) is split between the worker and the service

Reply:
You are right i have moved that to the service 👍

Reply:
🙌